zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -391,6 +400,7 @@ def _auto_divisibility(  # pylint: disable=line-too-long
     processing_chunk_sizes: Sequence[Vec3D[int]],
     processing_crop_pads: Sequence[Vec3D[int]],
     processing_blend_pads: Sequence[Vec3D[int]],
+    gap: Vec3D[int],  # pylint: disable=unused-argument
```

Comment:
Remove pylint directive

Reply:
Had to remove the whole line because, in fact, the argument was unused!